src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/AccessiblePackageIdentity.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Represents a package identity with an acessible name in the Preview Window Dialog in the PM UI
+    /// </summary>
+    public sealed class AccessiblePackageIdentity : PackageIdentity
+    {
+        private Lazy<string> _automationNameLazy;
+
+        public AccessiblePackageIdentity(string id, NuGetVersion version)
```

Comment:
Maybe I missed it, but [I only see the other constructor `AccessiblePackageIdentity(PackageIdentity)` being called](https://github.com/NuGet/NuGet.Client/pull/3217/files#diff-cf33c7d7a6896e312b8b86f1e05f557fR909-R914).

This constructor looks to be unnecessary.  If it is, remove it and move `_automationNameLazy` initialization into the other constructor.

Reply:
I implemented that constructor to avoid making lot of changes in which the original `PackageIdentity` constructor is used.

Reply:
Discussed offline. I realized that the changes are only on tests. So, I only need one constructor in production source code. I will remove the redundant constructor.